magento/pwa-studio
# Bug increase compatibility -- Bash style string compare should not be…

… used when environment is sh

## Description
See issue #1877

Changes [ "" == 0 ] to [ "" = "" ] the former will work for bash but the latter works on all sh terminals 
#1877


Closes #1877 

## Acceptance 

### Verification Stakeholders

### Specification

### Verification Steps
> docker/run-docker 
verify it does not die with "An error occured during setup." on Ubuntu 16.04

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
